build.sh

```diff
@@ -0,0 +1,51 @@
+name="oval-graph"
+module="oval_graph"
+
+git checkout master
+
+# Edit version, change log, etc... 
+vim ${module}/version.py
+vim ~/rpmbuild/SPECS/${name}.spec
+
+version=$(python3 setup.py --version)
+
+# Version commit
+git add -A
+git commit -m "${version}"
```

Comment:
You don't want to do `git add -A` in a script - it can pick up any trash that is present in the project dir. I would say don't even commit in the script, do the last commit manually, and when everything is OK, just tag.